docs/index.md

```diff
@@ -15,17 +15,21 @@ The RAPIDS Accelerator for Apache Spark combines the power of the <a href="https
 
```

Comment:
On a slightly unrelated note, should this be `README.md` instead of `index.md`?  I noticed Github isn't showing it by default when browsing the docs directory.

Reply:
I am not sure index.md is really a README outside the context of the rest of the pages.  It would not show a nav sidebar nor link to other pages in the repo (see [here](https://github.com/NVIDIA/spark-rapids/blob/branch-0.2/docs/index.md)).  We could add a separate README.md file to the /docs folder to point people to the doc site.  

As far as I can tell from https://pmarsceill.github.io/just-the-docs/docs/navigation-structure/ the parent page is index.md.